dist/ng-iban.js

```diff
@@ -308,39 +308,21 @@
           }
           return __modulo(parseInt(remainder, 10), 97) === 1;
         };
-        ctrl.$parsers.unshift(function(viewValue) {
+
+        ctrl.$validators.ngIban = function(viewValue) {
           var parsed, valid;
           if (viewValue != null) {
             valid = isValidIban(viewValue);
-            ctrl.$setValidity('iban', valid);
             if (valid) {
               parsed = parseIban(viewValue);
               if (parsed !== viewValue) {
                 ctrl.$setViewValue(parsed);
                 ctrl.$render();
               }
-              return parsed;
-            } else {
-              return void 0;
             }
```

Comment:
valid = false should be added here in order to avoid set the $validators to undefined.

`else{
   valid = false;
`